auth/server/src/main/java/com/seagates3/policy/IAMArnParser.java

```diff
@@ -0,0 +1,10 @@
+package com.seagates3.policy;
+
+public class IAMArnParser extends ArnParser {
+	public
+	IAMArnParser() {
+		
+	    this.regEx = "arn:aws:iam::" +
+                "[a-zA-Z0-9~@#$^*\\\\/_.-]*:[a-zA-Z0-9~@#$^*\\\\/_.-]+";
```

Comment:
resource arn format in IAM policy and Bucket policy different?

Reply:
yes.